DB/classification/models.py

```diff
@@ -57,6 +52,32 @@ def __init__(self, model_type="default", created_at=datetime.datetime.utcnow(),
         self.model = model
         self.id = _id
 
+    @classmethod
+    def reload_db_info(cls):
+        """
+        :summary: Reloads which database/user pair to use (for testing). Defaults to seads/seadapi when no file is present (old behavior)
+        """
+        db_info = None
+        try:
+            db_info = open("db_info", 'r')
```

Comment:
test_query_db is not a comparable test, since all of query_db's logic is not in actually using the database. All of its direct interaction with psycopg2 is in perform_query, which is not actually tested in test_query_db.

models.py, on the other hand, has all of its important logic in using psycopg2, which can't really be stubbed out without reimplementing most of the psycopg2 logic.

One option would be to pass the database information directly to models.py via the test, and to not pass that information through a config file. The test would still, however, construct and destroy a new database in the process of testing.